### ISSUE TITLE
[AzureRm.Automation.Netcore 0.11] - Import-AzureRmAutomationRunbook fails with DllNotFoundException

### ISSUE THREAD

**Issue by tiwood** [Azure/azure-powershell#5939]
<!--

If this issue is a bug report:
- Upgrade to the latest version of AzureRM and verify you are able to reproduce the issue
    - You can install the latest version of AzureRM from the PowerShell Gallery
        - https://www.powershellgallery.com/packages/AzureRM
    - You can also install the latest version from the Releases section
        - https://github.com/Azure/azure-powershell/releases
- Ensure that you repro the issue with $DebugPreference = "Continue" to receive the debug stream
- If this bug involves an exception being thrown, please run Resolve-AzureRmError to receive extended information on the error
- Fill out the below template

If this issue is not a bug report, please remove the below template

-->

### Description

Import-AzureRmAutomationRunbook fails with DllNotFoundException. I'm getting this error on macOS.

### Script/Steps for Reproduction

Try to import a Runbook to an existing Automation account

```powershell
$automation_account_name = "mytestautomationaccount"
$automation_account_rg = "mytestrg"
$runbook_name = "testrunbook"
$runbook_file_path = "./myrunbook.ps1"

$runbook_obj = Import-AzureRmAutomationRunbook -Path $runbook_file_path `
-Name $runbook_name `
-Type PowerShell `
-Published:$true `
-AutomationAccountName $automation_account_name `
-ResourceGroupName $automation_account_rg `
-Force
```

### Module Version

```
Version Name                                                                                                                                                                                                                                                                                                                                                                                                                             
------- ----                                  
0.11.0  AzureRM.Aks.Netcore                   
0.11.0  AzureRM.AnalysisServices.Netcore      
0.11.0  AzureRM.ApiManagement.Netcore         
0.11.0  AzureRM.ApplicationInsights.Netcore   
0.11.0  AzureRM.Automation.Netcore            
0.11.0  AzureRM.Backup.Netcore                
0.11.0  AzureRM.Batch.Netcore                 
0.11.0  AzureRM.Billing.Netcore               
0.11.0  AzureRM.Cdn.Netcore                   
0.11.0  AzureRM.CognitiveServices.Netcore     
0.11.0  AzureRM.Compute.Netcore               
0.10.0  AzureRM.Compute.Netcore               
0.11.0  AzureRM.Consumption.Netcore           
0.11.0  AzureRM.ContainerInstance.Netcore     
0.11.0  AzureRM.ContainerRegistry.Netcore     
0.11.0  AzureRM.DataFactories.Netcore         
0.11.0  AzureRM.DataFactoryV2.Netcore         
0.11.0  AzureRM.DataLakeAnalytics.Netcore     
0.11.0  AzureRM.DataLakeStore.Netcore         
0.11.0  AzureRM.DevTestLabs.Netcore           
0.11.0  AzureRM.Dns.Netcore                   
0.11.0  AzureRM.EventGrid.Netcore             
0.11.0  AzureRM.EventHub.Netcore              
0.11.0  AzureRM.Insights.Netcore              
0.11.0  AzureRM.IotHub.Netcore                
0.11.0  AzureRM.KeyVault.Netcore              
0.10.0  AzureRM.KeyVault.Netcore              
0.11.0  AzureRM.LogicApp.Netcore              
0.11.0  AzureRM.MachineLearning.Netcore       
0.11.0  AzureRM.MachineLearningCompute.Netcore
0.11.0  AzureRM.MarketplaceOrdering.Netcore   
0.11.0  AzureRM.Media.Netcore                 
0.11.0  AzureRM.Netcore                       
0.10.0  AzureRM.Netcore                       
0.11.0  AzureRM.Network.Netcore               
0.10.0  AzureRM.Network.Netcore               
0.11.0  AzureRM.NotificationHubs.Netcore      
0.11.0  AzureRM.OperationalInsights.Netcore   
0.11.0  AzureRM.PowerBIEmbedded.Netcore       
0.11.0  AzureRM.Profile.Netcore               
0.10.0  AzureRM.Profile.Netcore               
0.11.0  AzureRM.RedisCache.Netcore            
0.11.0  AzureRM.Relay.Netcore                 
0.11.0  AzureRM.Resources.Netcore             
0.10.0  AzureRM.Resources.Netcore             
0.11.0  AzureRM.Scheduler.Netcore             
0.11.0  AzureRM.ServiceBus.Netcore            
0.11.0  AzureRM.ServiceFabric.Netcore         
0.11.0  AzureRM.Sql.Netcore                   
0.11.0  AzureRM.Storage.Netcore               
0.10.0  AzureRM.Storage.Netcore               
0.11.0  AzureRM.StreamAnalytics.Netcore       
0.11.0  AzureRM.Tags.Netcore                  
0.10.0  AzureRM.Tags.Netcore                  
0.11.0  AzureRM.TrafficManager.Netcore        
0.11.0  AzureRM.UsageAggregates.Netcore       
0.11.0  AzureRM.Websites.Netcore              
0.10.0  AzureRM.Websites.Netcore          
```

### Environment Data

```powershell
Name                           Value                                                                                                                                                                                                                                                                                                                                                                                                    
----                           -----                                                                                                                                                                                                                                                                                                                                                                                                    
PSVersion                      6.0.2                                                                                                                                                                                                                                                                                                                                                                                                    
PSEdition                      Core                                                                                                                                                                                                                                                                                                                                                                                                     
GitCommitId                    v6.0.2                                                                                                                                                                                                                                                                                                                                                                                                   
OS                             Darwin 17.5.0 Darwin Kernel Version 17.5.0: Mon Mar  5 22:24:32 PST 2018; root:xnu-4570.51.1~1/RELEASE_X86_64                                                                                                                                                                                                                                                                                            
Platform                       Unix                                                                                                                                                                                                                                                                                                                                                                                                     
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                                                                                                                                                                                                                                                                                                                                  
PSRemotingProtocolVersion      2.3                                                                                                                                                                                                                                                                                                                                                                                                      
SerializationVersion           1.1.0.1                                                                                                                                                                                                                                                                                                                                                                                                  
WSManStackVersion              3.0                                                                                         
```

### Debug Output

```powershell
Import-AzureRmAutomationRunbook : Unable to load DLL 'api-ms-win-eventing-provider-l1-1-0.dll': The specified module or one of its dependencies could not be found.
 (Exception from HRESULT: 0x8007007E)
At line:1 char:16
+ ... nbook_obj = Import-AzureRmAutomationRunbook -Path $runbook_file_path  ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : CloseError: (:) [Import-AzureRmAutomationRunbook], DllNotFoundException
+ FullyQualifiedErrorId : Microsoft.Azure.Commands.Automation.Cmdlet.ImportAzureAutomationRunbook
 
DEBUG: AzureQoSEvent: CommandName - Import-AzureRmAutomationRunbook; IsSuccess - False; Duration - 00:00:00.0244210; Exception - System.DllNotFoundException: Unable to load DLL 'api-ms-win-eventing-provider-l1-1-0.dll': The specified module or one of its dependencies could not be found.
 (Exception from HRESULT: 0x8007007E)
   at System.Diagnostics.Eventing.UnsafeNativeMethods.EventActivityIdControl(Int32 ControlCode, Guid& ActivityId)
   at System.Diagnostics.Eventing.EventProvider.SetActivityId(Guid& id)
   at Microsoft.Azure.Commands.Automation.RequestSettings..ctor(IAutomationManagementClient automationClient)
   at Microsoft.Azure.Commands.Automation.Common.AutomationClient.ImportRunbook(String resourceGroupName, String automationAccountName, String runbookPath, String description, IDictionary tags, String type, Nullable`1 logProgress, Nullable`1 logVerbose, Boolean published, Boolean overwrite, String name)
   at Microsoft.Azure.Commands.Automation.Cmdlet.ImportAzureAutomationRunbook.AutomationProcessRecord()
   at Microsoft.Azure.Commands.Automation.Cmdlet.AzureAutomationBaseCmdlet.ExecuteCmdlet()
   at Microsoft.WindowsAzure.Commands.Utilities.Common.AzurePSCmdlet.ProcessRecord();
```

```powershell
Message        : Unable to load DLL 'api-ms-win-eventing-provider-l1-1-0.dll': The specified module or one of its dependencies could not be found.
                  (Exception from HRESULT: 0x8007007E)
StackTrace     :    at System.Diagnostics.Eventing.UnsafeNativeMethods.EventActivityIdControl(Int32 ControlCode, Guid& ActivityId)
                    at System.Diagnostics.Eventing.EventProvider.SetActivityId(Guid& id)
                    at Microsoft.Azure.Commands.Automation.RequestSettings..ctor(IAutomationManagementClient automationClient)
                    at Microsoft.Azure.Commands.Automation.Common.AutomationClient.ImportRunbook(String resourceGroupName, String automationAccountName, String runbookPath, String description, IDictionary tags, String type, Nullable`1 logProgress, Nullable`1 logVerbose, Boolean published, Boolean overwrite, String name)
                    at Microsoft.Azure.Commands.Automation.Cmdlet.ImportAzureAutomationRunbook.AutomationProcessRecord()
                    at Microsoft.Azure.Commands.Automation.Cmdlet.AzureAutomationBaseCmdlet.ExecuteCmdlet()
                    at Microsoft.WindowsAzure.Commands.Utilities.Common.AzurePSCmdlet.ProcessRecord()
Exception      : System.DllNotFoundException
InvocationInfo : {Import-AzureRmAutomationRunbook}
Line           :     $runbook_obj = Import-AzureRmAutomationRunbook -Path $runbook_file_path -Name $runbook_name `
                 
Position       : At /Users/andrebindewald/Repositories/dbg/dbg-azure-terraform/modules/custom-provisioners/pwsh-tfprov-manage-runbooks.ps1:135 char:20
                 + ... nbook_obj = Import-AzureRmAutomationRunbook -Path $runbook_file_path  ...
                 +                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
HistoryId      : 8
```

**Comment by cormacpayne** [comment]
@MiYanni would you mind taking a look at which this assembly is?

**Comment by MiYanni** [comment]
Reported here https://github.com/Azure/azure-powershell/issues/5858 but it seems we didn't fix it yet.

**Comment by MiYanni** [comment]
This will be fixed in the 0.12.0 build.

**Comment by kcighon** [comment]
Great news - what's the ETA for 0.12.0?

**Comment by MiYanni** [comment]
@kcighon Soon™️
But actually... probably next week. We are doing some //BUILD related activities at the moment.

**Comment by MiYanni** [comment]
@kc786 Just to give an update. I have the PR ready, but our team has to review it before we can make the 0.12.0 build. There were a number of issues and other things that I had to fix, so that's why it took longer than expected. You can track the PR here:
https://github.com/Azure/azure-powershell/pull/6281

**Comment by markcowl** [comment]
Fixed in the release candidate

**Comment by MiYanni** [comment]
@tiwood @kcighon Fixed in [0.12.0 for NetCore](https://github.com/Azure/azure-powershell/releases/tag/v0.12.0-nc-June2018)
